models/action.js

```diff
@@ -35,12 +37,14 @@ ActionSchema.statics.findById = function(id) {
  * @param {String} action the new action to the comment
  * @return {Promise}
  */
-ActionSchema.statics.insertUserAction = ({item_id, item_type, user_id, action_type}) => {
+ActionSchema.statics.insertUserAction = ({item_id, item_type, user_id, action_type, field, detail}) => {
```

Comment:
The alternative is to pass in a whole object, which seems messy (and frought, see @wyatt's comments below.) I'm leaving this as is.